hudi-client/hudi-client-common/src/main/java/org/apache/hudi/keygen/CommonComplexKeyGenerator.java

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
Those `CommonKeyGenerator`s can be used for both `flink` and `java` engine. 
I will add some flink specific `KeyGenerator`s  when flink implementation is  landed